src/legacy/core_plugins/vis_default_editor/public/components/agg.tsx

```diff
@@ -34,6 +34,7 @@ import { DefaultEditorAggCommonProps } from './agg_common_props';
 import { AGGS_ACTION_KEYS, AggsAction } from './agg_group_state';
 import { RowsOrColumnsControl } from './controls/rows_or_columns';
 import { RadiusRatioOptionControl } from './controls/radius_ratio_option';
+import { TimeRange } from '../../../../../plugins/data/common';
```

Comment:
wrong import. It should be: 
`import { TimeRange } from '../../../../../plugins/data/pulbic';`